llvm/lib/Target/AMDGPU/AMDGPUAttributor.cpp

```diff
@@ -1372,7 +1372,7 @@ static bool runImpl(Module &M, AnalysisGetter &AG, TargetMachine &TM,
   LLVM_DEBUG(dbgs() << "[AMDGPUAttributor] Module " << M.getName() << " is "
                     << (AC.IsClosedWorldModule ? "" : "not ")
                     << "assumed to be a closed world.\n");
-
+  uint32_t AddrSpaceMask = (1 << AMDGPUAS::MAX_AMDGPU_ADDRESS + 1) - 1;
```

Comment:
This should be `(1 << (AMDGPUAS::MAX_AMDGPU_ADDRESS + 1)) - 1` right?

Reply:
I don't understand how this can be a mask